mobile/salto/src/routes/+page.svelte

```diff
@@ -58,6 +83,11 @@
 
   const onAddNode = (nodeData: NodeData) => {
     nodes.push(nodeData);
+    let assets: string[] = totalBalance
+      .entries()
+      .map(([unit]) => (unit == "millistrk" ? "strk" : unit))
+      .toArray();
+    priceProviderAddAssets(assets);
```

Comment:
nooo. onAddNode is a front end reaction to a backend node addition.
It should just update the front.
here you are calling the backend back.
You were in the backend just before, when the node was added. Do what you need to do then, when you are already in the backend, reacting to user action.